src/entities/ShareClass.ts

```diff
@@ -1070,14 +1070,74 @@ export class ShareClass extends Entity {
     )
   }
 
+  investorOrders() {
+    const self = this
+
+    return this._query(['investorOrders'], () =>
+      self._investorOrders().pipe(
+        switchMap((orders) =>
+          combineLatest([
+            ...orders.outstandingInvests.map((order) => self._investorOrder(order.assetId, order.investor)),
+            ...orders.outstandingRedeems.map((order) => self._investorOrder(order.assetId, order.investor)),
+          ]).pipe(
+            switchMap((investorOrders) => {
+              const claimsByInvestor = new Map<
+                string,
+                {
+                  address: HexString
+                  assetId: AssetId
+                  maxRedeemClaims: number
+                  maxDepositClaims: number
+                }[]
+              >()
+
+              investorOrders.forEach((order) => {
+                const key = order.investor
+
+                if (claimsByInvestor.has(key)) {
+                  const existing = claimsByInvestor.get(key)!
+
+                  const existingForAsset = existing.find((e) => e.assetId.equals(order.assetId))
```

Comment:
`if (existingForAsset) return` 
they're literally the same order and shouldn't be summed